test/quantization/core/test_quantized_op.py

```diff
@@ -2917,6 +2917,11 @@ def test_advanced_indexing(self):
 
     @override_qengines
     def test_custom_module_lstm(self):
+        class QuantizedLSTMSplitGates(torch.ao.nn.quantizable.LSTM):
```

Comment:
for UX, will it be easier to reuse the default mapping: https://github.com/pytorch/pytorch/blob/e39955e82f494dcf6f22c0d81556e7cc26a281f7/torch/ao/quantization/quantize.py#L54 and just have user set some attributes on `nn.LSTM`?

Reply:
also this class should be called `QuantizableLSTMSplitGates` btw if you want to use this

Reply:
> just have user set some attributes on nn.LSTM

I thought about that but since we are not changing any behavior in `nn.LSTM` I was hesitant to change any code there. Definitely open to trying a cleaner way to toggle this if something comes up!